doc/content/devices/adding-devices/_index.md

```diff
@@ -55,63 +59,105 @@ See [ID and EUI constraints]({{< ref "reference/id-eui-constraints" >}}) for gui
 
 The device is now activated, and will appear as connected in {{% tts %}} once it sends an uplink.
 
-## Manually Create End Device
+## Manually Creating an End Device
 
 If your device is not in the device repository, you may manually register it.
 
 Make sure the **Manually** tab is selected.
 
-Choose the **Activation Mode**.
+{{< figure src="manual-tab.png" alt="Manual tab" >}}
 
-Select the device **LoRaWAN version**.
+Select the device **LoRaWAN version**. This should be provided with your device as the LoRaWAN version, LoRaWAN specification, or MAC version.
 
-If using Over-the-Air-Activation (OTAA) and an External Join Server, tick the **External Join Server** button.
+{{< note >}}
+If you can not find the LoRaWAN version of your device, use MACV1.0.2 and Regional Parameters PHY V1.0.2 Rev B, as these are the most common. But watch the [event log]({{< ref "getting-started/events" >}}) for errors!
```

Comment:
Do we really want to encourage this? We should at least say that using the wrong MAC/PHY versions can lead to complex issues that are hard to debug/diagnose. This is also the reason why we don't use default values here in the Console. cc @johanstokking

Reply:
If we don't do it, people pick a random value (probably 1.1) and then don't realize later why it isn't working. If we suggest this, people will at least make a more likely guess, and hopefully watch for errors.

Reply:
It's worse for users to make the mistake and choose a random value, than that they just thought they followed our instructions. Mentioning LoRaWAN 1.0.2 is as random as the user picking LoRaWAN 1.1. We can't really say anything about this at this point.

Like @kschiffer says, the activation will probably work, uplink messages will work too and downlink may work as well in most regions, but at some point in the future things will break if the LoRaWAN version is incorrect.

When the user is using advanced settings or manual adding the end device, they MUST know what LoRaWAN version they are using. Otherwise they are really in the wrong place at that moment.